source/grid/grid_tools.cc

```diff
@@ -364,6 +364,43 @@ namespace GridTools
   }
 
 
+
+  template <int dim, int spacedim>
+  BoundingBox<spacedim>
+  compute_bounding_box(const Triangulation<dim, spacedim> &tria)
+  {
+    using iterator = typename Triangulation<dim, spacedim>::active_cell_iterator;
+    const auto predicate = [](const iterator &)
+    {
+      return true;
+    };
+
+    return BoundingBox<spacedim>(compute_bounding_box(tria, predicate));
+  }
+
+
+
+  template <int dim, int spacedim>
+  BoundingBox<spacedim>
+  compute_bounding_box(const CellAccessor<dim, spacedim> &accessor)
```

Comment:
Why do you still need this? Can't you just call the function you implemented in `TriaAccessor`?